helix-term/src/commands.rs

```diff
@@ -1062,6 +1086,20 @@ mod cmd {
             fun: format,
             completer: None,
         },
+        Command {
+            name: "earlier",
+            alias: Some("ear"),
+            doc: "Jump back to an earlier point in edit history.",
+            fun: earlier,
+            completer: None,
+        },
+        Command {
+            name: "later",
+            alias: Some("lat"),
+            doc: "Jump to a later point in edit history.",
```

Comment:
I think we need to expand this doc and earlier to expand how to use the timings and numbers.
```
                                                                                             
 Jump back to an earlier point in edit history.                                              
                                                                                             
:earlier <cursor>
```
Not quite helpful here. I think we can just show that it can be two groups.

Reply:
Do you have a proposal?

Reply:
Maybe just "Accepts an optional number of steps or duration."? The space in the description box is limited. If we have documentation, we could add the detailed syntax there.

Reply:
Yes, but we currently don't have documentation. Even if we did stucking as much as we can into the box is still good because we don't need to go read documentation, the box is the documentation.

Reply:
Or at the very least show an example or two common ones (minutes and seconds should be fine + one step thing).

Reply:
Looks like we're limited here both horizontally and vertically. The box displays only one line (everything below a newline is cut). So before we're able to stick more info in there we'd need to change the code that displays it.

Reply:
Sure, maybe just keep a TODO here so I can follow up later on this.